atspi/examples/currently-focused-frame.rs

```diff
@@ -24,15 +24,13 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
 	let mut found_active_frame: bool = false;
 
-	for app in apps.iter() {
-		let proxy = app.clone().into_accessible_proxy(conn).await?;
+	for app in apps.into_iter().flat_map(|or| NonNullObjectRef::try_from(or).ok()) {
```

Comment:
On second thought.... this pattern is very common: continue if null, otherwise try to convert to proxy. That should probably just be a method?